Changing database port

Azmah-Bad opened the pull request:
made the port an environment variable

Azmah-Bad committed:
[PORT]: fixed #7
made the port an environment variable

Azmah-Bad committed:
[CI/CD] 🕳 added port as env variable

titouan-joseph commented on docker-compose.yml:
```diff
@@ -36,8 +36,10 @@ services:
+
     ports:
-      - 3306:3306 # for debug
+      - ${DATABASE_PORT}:${DATABASE_PORT} # for debug
```
I think it's better to use  ${DATABASE_PORT}:3306
To avode container bind problems

Azmah-Bad replied:
Keycloak database will not get exposed even for debug (unless extreme desperation) since it's already hard to read.

Azmah-Bad committed:
[PROD] 🎬 added production's docker compose
in production all we need to do now `docker-compose -f docker-compose.prod.yml`

Azmah-Bad committed:
Merge remote-tracking branch 'origin/database_port' into database_port
